tox.ini

```diff
@@ -1,13 +1,20 @@
 [tox]
-envlist = py{27,35,36}-celery{3,4},py37-celery4
+envlist =
+    py{27,35,36}-celery{3,4}
+    py37-celery4
+    py{27,35,36,37}-missing_modules
 skip_missing_interpreters = true
 
 [testenv]
+setenv =
+    missing_modules: TEST_MISSING_MODULES_HANDLING=1
```

Comment:
It looks like this is set unconditionally?

Reply:
No, it's [factor-conditional](https://tox.readthedocs.io/en/latest/config.html#factors-and-factor-conditional-settings).
`missing_modules` is a `factor` here.
This `setenv` is needed only for local `tox` usage because within the Travis CI environment `TEST_MISSING_MODULES_HANDLING` is set by Travis CI itself.